capreolus/benchmark/__init__.py

```diff
@@ -90,6 +89,76 @@ class Robust04Yang19(Benchmark):
     query_type = "title"
 
 
+@Benchmark.register
+class NF(Benchmark):
+    """ A Full-Text Learning to Rank Dataset for Medical Information Retrieval [1]
```

Comment:
We should also state which version of the qrels and topics we're using. I think this is 2-1-0 with all titles?

Reply:
oh I just added qrel range as one of the `Config` option, now both mode are supported